GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs

```diff
@@ -402,5 +404,16 @@ public void SortCalendarDates()
             this.ExecuteNonQuery("DROP TABLE calendar_date");
             this.ExecuteNonQuery("ALTER TABLE calendar_date_sorted RENAME TO calendar_date");
         }
+
+        /// <summary>
+        /// Deletes and recreates the polygons table in a sorted order (first by poly_pt_seq then by id) - may take time
+        /// </summary>
+        public void SortPolygons()//TODO: test!
+        {
+            this.ExecuteNonQuery("CREATE TABLE IF NOT EXISTS [polygons_sorted] ( [id] TEXT NOT NULL, [poly_pt_lat] REAL, [poly_pt_lon] REAL, [poly_pt_seq] INTEGER );");
+            this.ExecuteNonQuery("INSERT INTO polygons_sorted (id, poly_pt_lat, poly_pt_lon, poly_pt_seq) SELECT id, poly_pt_lat, poly_pt_lon, poly_pt_seq FROM polygons ORDER BY poly_pt_seq ASC, id ASC;");
```

Comment:
Wait, this doesn't look right. Surely it should be sorted by ID, then poly_pt_seq? Unless I'm going crazy. Did you test this sql line? We probably won't really need sort to be fair though

Reply:
I didn't really test it. I only added it as a safeguard in case something went wrong. Will test it now.

Reply:
seems you were right, but now something else broke in GTFSHelpers. will fix that now.